meteor/server/api/serviceMessages/serviceMessagesApi.ts

```diff
@@ -27,6 +27,13 @@ function readAllMessages(): Array<ServiceMessage> {
 	return messages
 }
 
+function convertExternalToServiceMessage(message: ExternalServiceMessage): ServiceMessage {
```

Comment:
It looks like this function is only ever used in the tests, perhaps it should be moved to `serviceMessagesApi.test.ts` instead?

Reply:
I moved the function to `serviceMessagesApi.test.ts`.